<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>601</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_signIn">
     <widget class="QLineEdit" name="edit_username">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
color: #000000;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string extracomment="username"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_password">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
color: #000000;</string>
      </property>
      <property name="text">
       <string extracomment="password"/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lb_username">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>USERNAME</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_password">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PASSWORD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_signIn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_verdict">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ff0000;</string>
      </property>
      <property name="text">
       <string>Invalid username or password</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_recoverFront">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>房间初始温度：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_recoverSpeed">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>310</y>
        <width>46</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>35</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_dashboard">
     <widget class="QPushButton" name="bt_tempDown">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DOWN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_tempUp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_temp">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_wdspd">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>风速</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_wdspdDown">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DOWN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_wdspdUp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_action">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>操作</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_state0">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关机</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_state1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开机</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_signOut">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>179</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>登出账户</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_settemp">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>25</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_setwdspd">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_temp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #050511;
color: #3333ff;</string>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_settemp">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #110f05;
color: #ffff33;</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_wdspd">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #051105;
color: #33ff33;</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_setwdspd">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #110005;
color: #ff3333;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_temp_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_settemp_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>设定温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_wdspd_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>风速</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_setwdspd_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>设定风速</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_fee">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_feeRate">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>费率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>230</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>费用：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>230</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>元</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>270</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>元</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>状态：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>模式：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_state">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>关机</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_mode">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>270</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>制冷</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>190</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>房间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_roomId">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>190</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>关机</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
